framework/PageToolbar/PageToolbar.css

```diff
@@ -33,3 +33,31 @@
 .pf-v5-c-toolbar__group .pf-m-toggle-group {
   margin-inline-end: 0;
 }
+
```

Comment:
What are these styles for? We should not have hub specific styling of the toolbar.

Reply:
moved to `./frontend/hub/my-imports/components/hub-import-collections-filters.css`